week-1/explore/objects.md

```diff
@@ -5,12 +5,19 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+it shows "f".
 
 Now enter just `console` in the Console, what output do you get back?
+console {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}
 
 Try also entering `typeof console`
+It says an object.
+
 
 Answer the following questions:
 
-What does `console` store?
+What does `console` store? It's store an Array.
```

Comment:
It's an object, not an array. An object containing a number of functions `f`

Reply:
Alright.